assets/questions/bomberman.js

```diff
@@ -0,0 +1,147 @@
+// Copyright 2017 The TIE Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Question data for Bomber man.
+ */
+
+globalData.questions['bomberman'] = {  // eslint-disable-line dot-notation
+  title: 'Bomberman',
+  starterCode: {
+    python:
+`def bomb(board):
+    return 0
+`
+  },
+  auxiliaryCode: {
+    python:
+`class AuxiliaryCode(object):
+    @classmethod
+    def negative(cls, board):
+        return -1
+    @classmethod
+    def ignoreWalls(cls, board):
+        if len(board) == 0 or len(board[0]) == 0: return 0
+        rows = len(board)
+        cols = len(board[0])
+        maxEnemy = 0
+        up = [0 for x in range(cols)]
+        left = 0
+        memo = [[0 for x in range(cols)] for y in range(rows)]
```

Comment:
Please add a more descriptive name than "memo". It's not clear what this is supposed to be.